scripts/ExportScripts/export_profile_monolithic.py

```diff
@@ -0,0 +1,55 @@
+from o3de.export_project import *
+from o3de import manifest
+import argparse
+import pathlib
+import os
+
+#Note: project_path and engine_path are computed before this script runs, and can be accessed via o3de_context.project_path
+#for now this script assumes an engine-centric approach, and we assume to use the source built engine for the engine_path
+
+#to use this script, go to the engine root directory, and issue the following command:
+#.\scripts\o3de.bat export-project -pp C:\path\to\project -es C:\path\to\export_profile_monolithic.py -nmp C:\path\to\build\non_mono -mp C:\path\to\build\mono -out C:\path\to\output -ll INFO
+
+parser = argparse.ArgumentParser(prog="Profile Monolithic exporter for windows",
+                                 description="Exports a profile monolithic packaged build for the project")
+parser.add_argument('-nmp', '--non-mono-build-path', type=pathlib.Path, required=True)
+parser.add_argument('-mp', '--mono-build-path',type=pathlib.Path, required=True)
+parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True)
+
+args = parser.parse_args(o3de_context.args)
+
+projects_path_cmake_option = f"-DLY_PROJECTS={str(o3de_context.project_path)}"
+
+project_json_data = manifest.get_project_json_data(project_path = o3de_context.project_path)
+project_name= project_json_data.get('project_name')
+
+
+#build non-monolithic artifacts
+process_command(['cmake', '-S', '.', '-B', args.non_mono_build_path, '-DLY_MONOLITHIC_GAME=0', projects_path_cmake_option], cwd= o3de_context.engine_path)
+
+process_command(['cmake', '--build', args.non_mono_build_path, '--target', 'Editor', '--target', 'AssetBundler', '--target', 'AssetProcessor', '--target', 'AssetProcessorBatch', '--config', 'profile'], cwd=o3de_context.engine_path)
+
+executables_dir = args.non_mono_build_path / 'bin' / 'profile'
+
+asset_processor_exe = executables_dir / 'AssetProcessorBatch.exe'
+
+asset_bundler_exe = executables_dir / 'AssetBundler.exe'
+
+process_command([asset_processor_exe, '--project-path', o3de_context.project_path], cwd=o3de_context.engine_path)
```

Comment:
The AssetProcessor can forward any `--regset` option values to the aggregate `bootstrap.game.*.setreg` files it builds.

The Command Line Argument documentation goes over the `--regset` option [here](https://www.o3de.org/docs/user-guide/settings/command-line-arguments/#command-line-support)

What this means, is that, you can use that option to inject into `bootstrap.game.<config>.setreg` via the [SetregBuilder](https://github.com/aws-lumberyard-dev/o3de/blob/ProjectExport/DraftReviewForMonolithicExportScript/Code/Tools/AssetProcessor/native/InternalBuilders/SettingsRegistryBuilder.cpp#L267-L276) any [Console Commands](https://www.o3de.org/docs/user-guide/settings/issue-az-console-commands/) that a user might need to run the application in a standalone.
For example the `bg_ConnectToAssetProcessor=false` option can be injected as follows to prevent the AssetProcessor from starting up on launch of the `${projectName}.GameLauncher`

```suggestion
process_command([asset_processor_exe, '--project-path', o3de_context.project_path, '--regset="/O3DE/Autoexec/ConsoleCommands/bg_ConnectToAssetProcessor=false"'], cwd=o3de_context.engine_path)
```

The generated `bootstrap.game.<config>.setreg` file in the `Cache/<asset-type>` folder for the project would then contain that setting.

Reply:
is it recommended to use this approach for specifying the start level to load in the regset as well? I'm tempted to do it that way, and require a parameter from the user, but it's difficult to tell if that's necessary, or just problematic, especially if the user has custom loading logic.